crates/symbolicator/src/metrics.rs

```diff
@@ -101,28 +101,26 @@ macro_rules! metric {
                 .send();
         })
     }};
-    (timer($id:expr), $block:block $(, $k:expr => $v:expr)* $(,)?) => {{
+
+    // we use statsd timers to send things such as filesizes as well.
+    (time_raw($id:expr) = $value:expr $(, $k:expr => $v:expr)* $(,)?) => {{
         use $crate::metrics::_pred::*;
-        let now = Instant::now();
-        let rv = {$block};
         $crate::metrics::with_client(|client| {
-            client.time_duration_with_tags($id, now.elapsed())
+            client.time_with_tags($id, $value)
                 $(.with_tag($k, $v))*
                 .send();
-        });
-        rv
+        })
     }};
 
-    // we use statsd timers to send things such as filesizes as well.
-    (time_raw($id:expr) = $value:expr $(, $k:expr => $v:expr)* $(,)?) => {{
+    // histograms
+    (histogram($id:expr) = $value:expr $(, $k:expr => $v:expr)* $(,)?) => {{
         use $crate::metrics::_pred::*;
         $crate::metrics::with_client(|client| {
-            client.time_with_tags($id, $value)
+            client.histogram_with_tags($id, $value)
                 $(.with_tag($k, $v))*
                 .send();
         })
     }};
```

Comment:
It looks like statsd [directly supports histograms](https://docs.rs/cadence/0.25.0/cadence/trait.Histogrammed.html#tymethod.histogram_with_tags).

The `metrics!` macro has been extended to include that functionality in this PR, but it appears that symbolicator is already emitting `StacktraceMetrics` as a histogram via `metrics!(timer_raw(...))`, which uses `time_with_tags`. 

Was there an explicit decision made not to use statsd's `histogram_with_tags()` for histograms?